lib/awful/spawn.lua

```diff
@@ -139,6 +139,58 @@ function spawn.with_line_callback(cmd, callbacks)
     return pid
 end
 
+--- Spawn a program and asynchronously and capture its output
```

Comment:
I'm not really good at English, but shouldn't it be: `Asynchronously spawn a program and capture its output`?